cvat/apps/dataset_manager/util.py

```diff
@@ -195,7 +195,7 @@ def parse_export_file_path(file_path: os.PathLike[str]) -> ParsedExportFilename:
         (
             r'(?P<export_mode>dataset|annotations)'
             r'(?:-instance(?P<instance_timestamp>\d+\.\d+))?' # optional for backward compatibility
-            r'-(?P<format_tag>.+)'
+            r'(?(instance_timestamp)-|_)(?P<format_tag>.+)'
```

Comment:
The conditional regex here seems unnecessary. Wouldn't it be possible to express the same thing as ```(?:-instance(?P<instance_timestamp>\d+\.\d+)-|_)```?

Reply:
Yes, seems to be working as well.